meson_options.txt

```diff
@@ -33,6 +33,7 @@ option('reduce-tolerance', type: 'boolean', value: true)
 option('enable-long-test', type: 'boolean', value: false)
 
 # backend options
+option('enable-encoder', type: 'boolean', value: true)
```

Comment:
We don't need to add enable-encoder here.

Reply:
Thank you for Review - Fix